pages/common/retry.md

```diff
@@ -0,0 +1,16 @@
+# retry
+
+> Repeat command until a criteria is met, usually success.
+> More information: <https://github.com/minfrin/retry>.
+
+- Retry a command until it succeeds:
+
+`retry {{command}}`
+
+- Retry a command every n seconds:
```

Comment:
I think the description should be more specific, the command only retry after failures.